_releases/3.12.0.md

```diff
@@ -0,0 +1,100 @@
+---
+version: 3.12.0
+releaseDate: 2019-01-31
+layout: release
+---
+
+## New Features <img width="25" alt ="features" src="https://user-images.githubusercontent.com/28867440/42755172-d8046d70-88ff-11e8-917c-ed1fcf2aac8c.png">
+
+- Invites and contract.availableRoles and tenantPolicy.roles. 
```

Comment:
`Invites and contract.availableRoles and tenantPolicy.roles` could we re-phrase it somehow?